lms/static/js/student_account/views/LoginView.js

```diff
@@ -105,10 +106,19 @@
                             anchorEnd: HtmlUtils.HTML('</a>')
                         });
 
-                    if (this.resetMessage.find('p').length === 0) {
-                        this.resetMessage.append(HtmlUtils.joinHtml(successMessage).toString());
-                    }
+                /* Temporarily remove these, then re-add them after the message
+                 * container has been shown. This seems to make them more visible
+                 * to screen readers in certain browsers (like IE 11).
+                 */
+                    $title.remove();
+                    $msg.remove();
                     this.element.show(this.$resetSuccess);
+                    this.$resetSuccess.append($title);
+                    this.$resetSuccess.append($msg);
+
+                    if ($msg.find('p').length === 0) {
```

Comment:
This was here previously, I just changed the variable it is using from `this.resetMessage` to `$msg`. I actually thought about getting rid of this altogether and replacing it with something like:
```
$msg.empty();
$msg.append(HtmlUtils.joinHtml(successMessage).toString());
```
I think that the existing implementation could have problems if, for instance, someone were to submit the password reset form twice with different emails. Thoughts?